test/upgrade/README.md

```diff
@@ -0,0 +1,83 @@
+# Upgrade Tests
+
+In order to get coverage for the upgrade process from an operator’s perspective,
+we need an additional suite of tests that perform a complete knative upgrade.
+Running these tests on every commit will ensure that we don’t introduce any
+non-upgradeable changes, so every commit should be releasable.
+
+This is inspired by kubernetes
+[upgrade testing](https://github.com/kubernetes/community/blob/master/contributors/devel/e2e-tests.md#version-skewed-and-upgrade-testing).
```

Comment:
This URL doesn't resolve, I assume https://github.com/kubernetes/community/blob/master/contributors/devel/sig-testing/e2e-tests.md was meant?